.github/workflows/release-drafter.yml

```diff
@@ -0,0 +1,33 @@
+name: Release drafter
+
+# Push events to every tag not containing "/"
+on:
+  push:
+    tags:
+      - "*"
+
+jobs:
+  draft-a-release:
+    name: Draft a release
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - id: get_data
+        run: |
+          echo "approvers=$(cat .github/CODEOWNERS | grep @ | tr -d '*\n ' | sed 's/@/,/g' | sed 's/,//1')" >> $GITHUB_OUTPUT
+          echo "version=${GITHUB_REF#refs/tags/}" >> $GITHUB_OUTPUT
+      - uses: trstringer/manual-approval@v1
+        with:
+          secret: ${{ github.TOKEN }}
+          approvers: ${{ steps.get_data.outputs.approvers }}
+          minimum-approvals: 2
+          issue-title: 'Release opensearch-migrations version ${{ steps.get_data.outputs.version }}'
+          issue-body: "Please approve or deny the release of opensearch-migrations **TAG**: ${{ github.ref_name }}  **COMMIT**: ${{ github.sha }} **VERSION** : ${{ steps.get_data.outputs.version }} "
+          exclude-workflow-initiator-as-approver: true
+
+      - name: Draft a release
+        uses: softprops/action-gh-release@v1
+        with:
+          draft: true
+          generate_release_notes: true
+          files: artifacts.tar.gz
```

Comment:
We need to first create this tar.gz before uploading. Can you please try out this workflow in your fork?
Thanks!

Reply:
Sure Sayali, will test it.

Reply:
Hey @gaiksaya with the workflow I have able to create a draft release upon a tag (0.3.0) push.
<img width="1338" alt="Screenshot 2023-10-04 at 1 59 28 PM" src="https://github.com/opensearch-project/opensearch-migrations/assets/28733332/39416489-c089-4c7f-bb06-6076863df3c8">